wordpress/docker-compose.yml

```diff
@@ -23,6 +23,7 @@ x-wordpress: &wordpress_app
     - ${WEB_ACCESS_FOLDER}:/var/www/html/docs
     - ${METADATA_FOLDER}:/var/www/html/metadata
     - ${PROFILE_FOLDER}:/var/www/html/profile
+    - ${GOACCESS_FOLDER}:/var/www/html/goaccess
```

Comment:
we haven't used goaccess (html reports that can be generated weekly), and are moving towards a more robust reporting software, so can avoid adding this.